### PR TITLE
Fix a bug when a document has an empty body

### PR DESCRIPTION
**Pull request by gregkare**
Add check on content length for an empty file

Refs #48

### CODE REVIEW

**Comment by gregkare** [comment]
So far I haven't been able to reproduce the issue in a test. The Riak Ruby client (at least the version we're using) cannot store nil, and the issue doesn't occur with an empty file (the data is an empty string, not nil)

This fixes the issue though.

File: lib/remote_storage/riak.rb [hunk 1]
```diff
@@ -68,7 +68,8 @@ def get_data(user, directory, key)
       when "application/json"
         return object.data.to_json
       else
-        return serializer_for(object.content_type) ? object.data : object.raw_data
+        # Never return nil, always turn data into a string
```
Comment (michielbdejong) [inline lib/remote_storage/riak.rb]: Not sure how strings work in Ruby, but if it's the same as in JavaScript, then turning the raw data into a string would corrupt binary data where not all bytes can necessarily be interpreted as utf-8 characters.

Comment (gregkare) [inline lib/remote_storage/riak.rb]: Yeah that fix doesn't make sense, thanks. I completely forgot about this pull request. I'm going to do a nil check and return an empty string only when needed.

**Comment by gregkare** [comment]
The crash also happens in the web server, so it can't be tested in a test. @skddc @galfert can you review and merge? Thanks!

**Comment by skddc** [comment]
+1

**Pull request merged**